### PR TITLE
AUTH-5608 added warp auth fields to access organizations and apps

### PR DESCRIPTION
**Pull request by russellkerns**
## Has your change been tested?

Unit tests

## Types of changes

What sort of change does your code introduce/modify?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: .changelog/1496.txt
Comment (jacobbednarz) [inline .changelog/1496.txt]:
```suggestion
```release-note:enhancement
access_application: Add support for allow_authenticate_via_warp
```

```release-note:enhancement
access_organization: Add support for `allow_authenticate_via_warp` and `warp_auth_session_duration`
```
```

File: access_application.go
Comment (jacobbednarz) [inline access_application.go:55]: nit: s/Warp/WARP

**Review by jacobbednarz — APPROVED** [review]
lgtm, ping me once the docs land and we can :ship:

**Comment by russellkerns** [comment]
> lgtm, ping me once the docs land and we can 🚢

This was just released in the docs!

**Pull request merged**
